client/src/components/TypeList.vue

```diff
@@ -1,13 +1,13 @@
 <script lang="ts">
-import { defineComponent, reactive } from '@vue/composition-api';
+import { computed, defineComponent, reactive } from '@vue/composition-api';
 import {
   useCheckedTypes, useAllTypes, useTypeStyling, useHandler,
 } from '../provides';
 
 export default defineComponent({
   name: 'TypeList',
-
-  setup() {
+  setup(_props, ctx) {
+    const prompt = ctx.root.$prompt;
```

Comment:
```suggestion
  setup(_, { root }) {
    const prompt = root.$prompt;
```

Restrict to only necessary params.